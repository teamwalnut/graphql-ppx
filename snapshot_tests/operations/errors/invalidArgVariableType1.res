/**
 * Required argument error should take prioroty over type error
 * Expecting this to be a required related error, not a type one
 */
module MyQuery = %graphql(`
  query invalidArgValues {
    invalidArgValues(
        stringRequired: $arg
    )
  }
`)
