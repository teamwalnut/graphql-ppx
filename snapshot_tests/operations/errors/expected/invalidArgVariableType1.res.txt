
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/invalidArgVariableType1.res[0m:[2m8:25-28[0m

   6 [2mâ”‚[0m   query invalidArgValues {
   7 [2mâ”‚[0m     invalidArgValues(
   [1;31m8[0m [2mâ”‚[0m         stringRequired: [1;31m$arg[0m
   9 [2mâ”‚[0m     )
  10 [2mâ”‚[0m   }
  11 [2mâ”‚[0m `)

  Invalid argument. The variable "$arg" on argument "stringRequired" is missing.

